src/rpc/mining.cpp

```diff
@@ -814,11 +814,9 @@ static RPCHelpMan getblocktemplate()
         time_start = GetTime();
 
         // Create new block
-        CScript scriptDummy = CScript() << OP_TRUE;
-        block_template = miner.createNewBlock(scriptDummy);
+        block_template = miner.createNewBlock(CScript() << OP_TRUE);
```

Comment:
In commit b35b23d823ac0fc43e4d137525167a7788ad672a:

Nit: I think we just directly call the new function here?

Reply:
This change was premature since the interface is changed in a later commit. It also doesn't do anything, so I'm dropping it.